### PR TITLE
viewmodel added

### PR DESCRIPTION
**Pull request by adamheinrich-oke**
Added viewmodel for search title fragment added two packages details and search to split the files.

### CODE REVIEW

File: app/src/main/java/com/example/adamh_miniapp/screens/search/SearchTitleViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+class SearchTitleViewModel : ViewModel() {
+    init {
+        Log.i("SearchTitleViewModel", "ViewModel created!")
+    }
+
+    override fun onCleared() {
+        super.onCleared()
+        Log.i("SearchTitleViewModel", "ViewModel destroyed!")
+    }
+
```
Comment (OKEMateuszKrawczuk) [inline app/src/main/java/com/example/adamh_miniapp/screens/search/SearchTitleViewModel.kt:15]: Ditto

File: app/src/main/java/com/example/adamh_miniapp/screens/search/SearchTitleFragment.kt [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
+import com.example.adamh_miniapp.R
 import com.example.adamh_miniapp.databinding.FragmentSearchTitleBinding
 
 class SearchTitleFragment : Fragment() {
 
+    private lateinit var viewModel: SearchTitleViewModel
+
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                               savedInstanceState: Bundle?): View? {
+
```
Comment (OKEMateuszKrawczuk) [inline app/src/main/java/com/example/adamh_miniapp/screens/search/SearchTitleFragment.kt:19]: Whitespace

**Review by OKEMateuszKrawczuk — CHANGES_REQUESTED** [review]

File: .idea/vcs.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (AgataZiolkowska) [inline .idea/vcs.xml:1]: Please add a proper .gitignore file

File: app/src/main/java/com/example/adamh_miniapp/screens/search/SearchTitleViewModel.kt (hunk 1 above)
Comment (AgataZiolkowska) [inline app/src/main/java/com/example/adamh_miniapp/screens/search/SearchTitleViewModel.kt:15]: empty space ;P

File: app/src/main/java/com/example/adamh_miniapp/screens/search/SearchTitleViewModel.kt [hunk 4]
```diff
@@ -0,0 +1,16 @@
+package com.example.adamh_miniapp.screens.search
+
+import android.util.Log
+import androidx.lifecycle.ViewModel
+
+class SearchTitleViewModel : ViewModel() {
+    init {
```
Comment (AgataZiolkowska) [inline app/src/main/java/com/example/adamh_miniapp/screens/search/SearchTitleViewModel.kt:7]: missing space

File: app/src/main/java/com/example/adamh_miniapp/screens/search/SearchTitleFragment.kt [hunk 5]
```diff
@@ -1,19 +1,27 @@
...
         val binding = DataBindingUtil.inflate<FragmentSearchTitleBinding>(inflater,
             R.layout.fragment_search_title,container,false)
+
+        viewModel = ViewModelProvider(this).get(SearchTitleViewModel::class.java)
```
Comment (AgataZiolkowska) [inline app/src/main/java/com/example/adamh_miniapp/screens/search/SearchTitleFragment.kt:23]: Could be joined to the initialization?

File: .idea/vcs.xml (hunk 3 above)
Comment (OKEMateuszKrawczuk) [reply .idea/vcs.xml:1]: You can find nice examples on gitignore.io

**Review by OKEMateuszKrawczuk — APPROVED** [review]

**Review by AgataZiolkowska — APPROVED** [review]

**Pull request merged**